core/src/main/java/com/klaytn/caver/utils/SecureRandomUtils.java

```diff
@@ -18,13 +18,13 @@
  * Modified and improved for the caver-java development.
  */
 
-package com.klaytn.caver.wallet;
+package com.klaytn.caver.utils;
```

Comment:
Is SecureRandomUtils a new implementation recently?

Reply:
No, It exists com.klaytn.caver.wallet package.
Previously, this class was used only in the Wallet.java file.(generateRandonBytes())
The reason I moved this class to the Utils package is because I use a lot of functions that generate random numbers from multiple packages.

Reply:
Um, then i think this should not moved and random generate function should be implemented separately in utils package. 
Maybe user might use this in their code. What's your thinking?

Reply:
generateRandomBytes() in Wallet class has "default" access modifier.(SecureRandomUtils class also has "default" access modifier.)
It means, user can't access generateRandomBytes() in Wallet class.
Because user will use it in an external package, not a wallet package.

So, I think it is better that move this method and SecureRandom to utils package.

Reply:
Oh, i didn't check the modifier. okayokay thank you for explaining